docs/SOPs/dataset_wrangling_SOP.md

```diff
@@ -120,9 +120,21 @@ After the spreadsheet is generated some manual steps can help contributors under
 
 Once you have a customised and potentially pre-filled spreadsheet it can be sent to the contributor along with the contributor spreadsheet guide. It is generally an iterative process of the contributor filling in what they can, the wrangler reviewing, curating and asking questions before further curation until the metadata is complete. 
 
-## Raw data upload (fastq)
+## Raw Data Upload (fastq)
 
-In order for a contributor to upload their data, you will need to provide them with a data upload area UUID as well as a set of contributor AWS access keys. 
+In order for a contributor to upload their data, you will need to:
+
+- provide them with a data upload area UUID
+- create an AWS user for them
+  
+```shell
```

Comment:
It should probably state that a wrangler can't run this process on its own, so we need to come to a dev to do this